include/openssl/prov_ssl.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2021 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
since you're importing content from old files, maybe use the oldest copyright?  I am not a lawyer.